ricecooker/commands.py

```diff
@@ -3,10 +3,17 @@
 import requests
 import json
 import webbrowser
-from ricecooker import config
-from ricecooker.classes import nodes, questions
+from . import config
+from .classes import nodes, questions
 from requests.exceptions import HTTPError
-from ricecooker.managers import ChannelManager, RestoreManager, Status
+from .managers import ChannelManager, RestoreManager, Status
+
+# Fix to support Python 2.x.
+# http://stackoverflow.com/questions/954834/how-do-i-use-raw-input-in-python-3
+try:
+    input = raw_input
+except NameError:
+    pass
```

Comment:
I hadn't realized at the time I added this that there was a decision to give up on 2.x support. If we don't plan to try to support 2.x at all, I can remove this.